.run/Infection on entire Project.run.xml

```diff
@@ -0,0 +1,6 @@
+<component name="ProjectRunConfigurationManager">

```

Comment:
These `.run` files come from your IDE, and shouldn't be part of the PR

Reply:
Yes, they came from PhpStorm. I'm removing them!